runtime/queries/cpp/highlights.scm

```diff
@@ -12,13 +12,6 @@
 (template_method
   name: (field_identifier) @function)
 
-(template_function
-  name: (identifier) @function)
```

Comment:
Without this stanza, template function calls are highlighted as variables

```cpp
a<T>();
```

Reply:
I removed this and the other stanza because they are duplicates of existing ones in the same file, and their removal didn't seem to make any difference. But if the duplicates actually do make a difference I'll put them back

Reply:
Oh whoops I didn't see the duplicates 😅. Yeah these are ok to remove 👍